src/client/transport.js

```diff
@@ -65,6 +65,11 @@ class Transport {
             const msgCallback = function () {
                 transport.activeServiceMessagesCounter--;
 
+                if (nativeMethods.xhrStatusGetter.call(this) !== 200 && this.responseText) {
+                    errorHandler.call(this); // eslint-disable-line no-use-before-define
```

Comment:
What's about avoiding this ESLint rule hack by extracting `errorHandler` and `msgCallback` into separate class methods? We can name them e.g. `handleMsgError` and `handleMsgData`.